src/main/java/com/example/gildedrose/item/GildedRose.java

```diff
@@ -12,4 +12,8 @@ public GildedRose(ArrayList<Updateable> items) {
     public void updateQuality() {
         items.forEach(Updateable::update);
     }
+
+    public ArrayList<Updateable> getItems() {
+        return this.items;
+    }
```

Comment:
**question**  Why is it necessary to add a getter to the GildedRose class?

What are some other ways we could accomplish our goal without expanding the interface of this class? Does this violated any SOLID principles?

Reply:
I suppose we don't need the getter if we make items public and pass the items to GildedRoseService.

Reply:
**thought**: Would making `items` public expose any risks down the line? Years from now could I, as a new developer on the project with a big deadline, cause accidental chaos with something like the below? 

`GildedRose.items = new ArrayList<Updateable>(Arrays.asList(new IHaveABigDeadlineTodayItem()))` 

There's perhaps some opportunities for encapsulation here 😸